easybuild/easyconfigs/s/SPAdes/SPAdes-3.13.0-foss-2018b.eb

```diff
@@ -0,0 +1,46 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+# 3.9.0:
+# Modified by:
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = "CMakeMake"
+
+name = 'SPAdes'
+version = '3.13.0'
+
+homepage = 'http://cab.spbu.ru/software/spades/'
+description = """Genome assembler for single-cell and isolates data sets"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['http://cab.spbu.ru/files/release%(version)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['c63442248c4c712603979fa70503c2bff82354f005acda2abc42dd5598427040']
+
+builddependencies = [('CMake', '3.12.1')]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('bzip2', '1.0.6'),
+    ('libreadline', '7.0'),
+]
+
+start_dir = 'src'
+
+separate_build_dir = True
+
+configopts = ' -DBoost_NO_BOOST_CMAKE=ON'
+
+sanity_check_commands = [('spades.py', '--test')]
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['spades-bwa', 'spades-hammer',
+                                     'spades-ionhammer', 'spades.py']],
```

Comment:
there are quite a few more things in `bin`, and at least `spades-core` looks important (?), should this list be updated/extended?